src/js/components/DataTable/DataTable.js

```diff
@@ -206,6 +208,19 @@ const DataTable = ({
     if (nextScrollOffset !== scrollOffset) setScrollOffset(nextScrollOffset);
   });
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I feel like we've gotten in trouble for doing this in other places and am wondering if we find a way to avoid this.

Reply:
Yup. I'll look at this more. In this case I do want it run after every render and it will only cause a re-render if it has to change the min height. Dependencies on this prevent that causing the size to be wrong on a resize. I think that is also true and existing defect with the existing layout effect that is updating the placeholder.top and .bottom based on the header and footer.